src/IntegrationUtils.ts

```diff
@@ -417,3 +427,28 @@ function validateDependency(text : string) {
 	}
 	return null;
 }
+
+function getChildProcessForIntegration(integration:string) {
+	if (childProcessMap) {
+		if (childProcessMap.has(integration)) {
+			return childProcessMap.get(integration);
+		}
+	}
+	return null;
+}
+
+// this doesn't work, but it's on the right track I think
+export function killChildProcessForIntegration(integration:string) : Promise<boolean> {
+	return new Promise( (resolve, reject) => {
+		let childProcess : any = getChildProcessForIntegration(integration);
+		if (childProcess) {
+			try {
+				childProcess.kill("SIGKILL");
```

Comment:
we really need the SIGKILL? If yes, it means that there is a problem in kamel which is ignoring the SIGINT or SIGTERM.

Reply:
Turns out we don't need SIGKILL now. Removed it.